raw_data/preprocessed/africa/botswana_english.txt

```diff
@@ -0,0 +1,107 @@
+
+Application of Act
+(3) [1] This Act applies to the activities in respect of which the Minister, may, after screening them, prescribe by regulations. [2] Regulations made pursuant to subsection [1] shall prescribe [a] a list of activities which are likely to cause significant adverse effects on the environment, or the locations that may be environmentally sensitive, in respect of which a statement shall be mandatory; [b] threshold determinations of environmental impact assessment with respect to the activities prescribed under paragraph [a]; and [c] criteria which shall be used to determine the likely effects of a proposed activity in order to further determine whether or not a statement is required for the activity. [3] In this section, "screening" means an initial stage in the environmental impact assessment process where the Minister determines whether an activity should be subjected to an environmental impact assessment or not.
+
+Requirements to undertake an activity
+(4) [1] No person shall undertake or implement an activity prescribed under section 3 unless the environmental impact of the proposed activity is fully taken into account in accordance with the provisions of this Act and authorisation has been issued under section [14]. [2] Any person who undertakes or implements an activity in contravention of subsection [1] commits an offence and is liable to a fine not exceeding P100000 or to a term of imprisonment not exceeding two years or to both. [3] A person who contravenes subsection [1] shall rehabilitate the area affected by the adverse environmental impact of the implemented activity. [4] In this section, "rehabilitate" means the reinstatement or restoration to a normal or functional environmental state. [5] A person who contravenes subsection [3] commits an offence and is liable to a fine not exceeding P15000 or to a term of imprisonment not exceeding 2 years or to both. [6] In the case of a continued contravention in respect of a person convicted under subsection [2] or [5], that person is liable to a further fine not exceeding P2000 for every day on which the offence continues.
```

Comment:
`section [14]` should read `section 14`. The rule, which I should have made clearer earlier (sorry) is that references to sub-article delimiters should be encapsulated in square brackets, while references to article numbers should not.